Rakefile

```diff
@@ -32,6 +32,11 @@ end
 desc "Run tests and style checker"
 task test: %w[spec rubocop]
 
+desc 'Check for unapproved licenses in dependencies'
+task(:license_finder) do
+  system('license_finder --decisions-file=.dependency_decisions.yml') || raise(StandardError, 'Unapproved license(s) found on dependencies')
```

Comment:
Is there a way for the error to specify which dependencies/licenses are a problem?

Reply:
The output of `license_finder` includes that information, so it will be printed anyway.